openstudiocore/resources/analysis/pat_export/scripts/StandardReports/measure.rb

```diff
@@ -1,139 +1,165 @@
 require 'erb'
+require 'json'
+
+require "#{File.dirname(__FILE__)}/resources/os_lib_reporting"
+require "#{File.dirname(__FILE__)}/resources/os_lib_schedules"
+require "#{File.dirname(__FILE__)}/resources/os_lib_helper_methods"
 
 #start the measure
-class StandardReports < OpenStudio::Ruleset::ReportingUserScript
+class OpenStudioResults < OpenStudio::Ruleset::ReportingUserScript
```

Comment:
@macumber it needs 1.7.5 or later I would have had to add some code to support pre 8.3 E+ due to SQL query changes.